ingestion/functions/parsing/riograndedosul/riograndedosul_test.py

```diff
@@ -0,0 +1,48 @@
+import csv
+import json
+import os
+import tempfile
+import unittest
+from datetime import date
+
+import pytest
+
+from riograndedosul import riograndedosul
+
+_SOURCE_ID = "abc123"
+_SOURCE_URL = "https://ti.saude.rs.gov.br/covid19/download"
+_PARSED_CASE = (
+    {
+        'caseReference': {
+            'sourceId': "abc123",
+            'sourceUrl': "https://ti.saude.rs.gov.br/covid19/download"
+        },
+        'location': {
+            'query': 'Água Santa, Rio Grande do Sul, Brazil'
+            },
+        'events': [
+            {
+                'name': 'confirmed',
+                'dateRange': {'start': '08/03/2020Z', 'end': '08/03/2020Z'}
+            },
+            {
+                'name': 'onsetSymptoms',
```

Comment:
perhaps tune the test data to cover more cases (hospotalization/outcome)

Reply:
I am so glad you asked me to do this, uncovered some mismatched indices that I hadn't caught otherwise